koku/reporting/provider/__init__.py

```diff
@@ -1 +1,40 @@
 # noqa
+from django.db.models import Field
+from django.db.models import Lookup
+
+
+# Custom Django Lookups
+@Field.register_lookup
+class AllowNullIcontains(Lookup):
+    """
+    This custom lookup accepts a list as the param wne will create
+    a NOT LIKE with an 'AND' between each element in the list.
+    Currently exclusively beinng used for tag exclusions.
+    """
+
+    lookup_name = "notlikelist"
```

Comment:
Is this lookup supposed to be the opposite of `icontains`? Should we just call this `noticontains`?

Reply:
I had it originally call noticontains, then renamed it after much debate since this one takes a list. I am okay with going back to icontains.

Reply:
I wasn't sure why it was named this, which is why I asked. I was only advocating for changing it if it was supposed to be the opposite of `icontains`. If it is a different lookup entirely, then let's name it whatever we want.